bibliotheca-hertziana-max-planck-institute-for-art-history.csl

```diff
@@ -0,0 +1,317 @@
+                </group>
+                <text macro="publisher" prefix=" " suffix=","/>
+              </group>
+              <group delimiter=" ">
+                <text variable="publisher-place" suffix=","/>
+                <date date-parts="year" form="text" variable="issued" suffix=","/>
+              </group>
+              <group>
+                <label strip-periods="true" suffix=". " variable="page" form="short"/>
+                <text variable="page" suffix="."/>
+              </group>
+              <group>
+                <text variable="URL" prefix="URL: " suffix="."/>
+              </group>
+            </group>
+          </else-if>
+          <else-if type="paper-conference" match="any">
+            <group delimiter=" ">
+              <group>
+                <group delimiter=" ">
+                  <text term="in" text-case="lowercase"/>
+                  <text variable="container-title" font-style="italic"/>
+                </group>
+                <text macro="edition"/>
+                <text macro="editor"/>
+                <group>
+                  <text variable="number-of-volumes" form="short" suffix=" vols.,"/>
+                </group>
+                <text macro="publisher" prefix=" "/>
+              </group>
+              <group delimiter=" ">
+                <date date-parts="year" form="text" variable="issued" suffix=", "/>
+              </group>
+              <group delimiter=" ">
+                <label strip-periods="true" suffix=". " variable="page" form="short"/>
+                <text variable="page" suffix="."/>
+              </group>
+              <group>
+                <text variable="URL" prefix="URL: " suffix="."/>
+              </group>
+            </group>
+          </else-if>
+          <else-if type="thesis" match="any">
+            <group delimiter=" ">
+              <group delimiter=", ">
+                <group>
+                  <text value="Ph.D Thesis"/>
+                </group>
+                <text macro="publisher" prefix=" "/>
+              </group>
+              <date date-parts="year" form="text" variable="issued" suffix="."/>
+              <group>
+                <text variable="URL" prefix=" URL: " suffix="."/>
+              </group>
+            </group>
+          </else-if>
+          <else-if type="manuscript" match="any">
+            <group delimiter=" ">
+              <group delimiter=", ">
+                <group>
+                  <text value="Manuscript"/>
+                </group>
+                <text macro="publisher" prefix=" "/>
+              </group>
+              <date date-parts="year" form="text" variable="issued" suffix="."/>
+              <group>
+                <text variable="URL" prefix=" URL: " suffix="."/>
+              </group>
+            </group>
+          </else-if>
+          <else>
+            <group delimiter=" ">
+              <text value="This reference type is not currently supported"/>
```

Comment:
we'd much rather avoid this. CSL is written so as to allow sensible fallbacks for item types not covered by a citation style.

Reply:
removed